tests/unit/Actions.spec.ts

```diff
@@ -285,7 +285,7 @@ describe.skip('Emits error event if NR validation fails in file and pay', () =>
     const localVue = createLocalVue()
     localVue.use(VueRouter)
     const router = mockRouter.mock()
-    router.push({ name: 'review-confirm', query: { id: 'T1234567' } })
+    await router.push({ name: 'review-confirm', query: { id: 'T1234567' } })
```

Comment:
The test worked without this, but technically, "push()" returns a promise, so this call should be awaited.